lib/pure/times.nim

```diff
@@ -172,30 +178,43 @@ proc getDaysInYear*(year: int): int =
   ## Get the number of days in a ``year``
   result = 365 + (if isLeapYear(year): 1 else: 0)
 
-proc assertValidDate(monthday: MonthdayRange, month: Month, year: int) {.inline.} =
-  assert monthday <= getDaysInMonth(month, year),
-    $year & "-" & $ord(month) & "-" & $monthday & " is not a valid date"
+proc assertValidDate(d: Date) {.inline.} =
+  assert d.monthday <= getDaysInMonth(d.month, d.year),
+    $d.year & "-" & $ord(d.month) & "-" & $d.monthday & " is not a valid date"
+
+proc toEpochDate*(time: Time): EpochDate =
+  EpochDate(time.int64 div secondsInDay)
+
+proc toEpochDate*(epoch_days: int32): EpochDate =
```

Comment:
Should be `epochDays` instead of `epoch_days`

Reply:
Done